.flake8

```diff
@@ -15,5 +15,4 @@ max-complexity = 10
 
 # List ignore rules one per line.
 ignore =
-    E501
```

Comment:
https://www.flake8rules.com/rules/E501.html is there a another code for slightly longer?

Reply:
E501 is customisable by setting eg `max-line-length = 127` if we want to do that.